src/content/docs/en/community-resources/talks.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Talks, Interviews, and Streams
+description: Watch conference talks, interviews, and live streams about Astro!
+i18nReady: true
+---
+
+import Badge from '~/components/Badge.astro';
+
+Want to watch/listen to stuff? Some good things are listed here. For even more updated regularly, see this YouTube playlist, read our blog, subscribe to our monthly newsletter...
```

Comment:
```suggestion
Do you prefer learning by watching or listening? Many awesome members of the Astro community have produced informative and entertaining media that you can learn from and enjoy! For even more updated regularly, see this YouTube playlist, read our blog, subscribe to our monthly newsletter...
```
Possible sentence to put here? Left the last sentence to be replaced later (assuming that is the intention)!